docs/reference/analysis/analyzers/stop-analyzer-language-support.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[analysis-stop-analyzer-language-support]]
+=== Stop Analyzer Language Support
+
+The `stop` analyzer supports the following languages:
+
+[width="100%"]
+|=======
+|Language |Source

```

Comment:
I think it may be helpful to link to the stopwords list in Lucene where applicable. I included an example for Arabic below.

Reply:
Understood. I will link to the proper line indicating the stopwords list.